### PR TITLE
Update roadmap

### PR DESCRIPTION
**Pull request by karlchillmaid**
This pull request updates the roadmap page to reflect:

* what we’re working on now
* what we’re doing next
* what we’ve done

### CODE REVIEW

File: app/templates/views/guidance/features/roadmap.html [hunk 1]
```diff
@@ -27,28 +27,30 @@ <h1 class="heading-large">Roadmap</h1>
...
+    <li>Test Rich Communication Services (RCS)</li>
+    <li>Explore alternative letter suppliers</li>
   </ul>
 
   <h2 class="heading-medium" id="things-we-will-do-later">Things we’ll do later</h2>
 
   <ul class="govuk-list govuk-list--bullet">
-    <li>Find ways to help services write better notifications</li>
-    <li>Explore new channels for notifications (<a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/service-manual/agile-delivery/how-the-alpha-phase-works">alpha</a>)</li>
+    <li>Launch RCS messaging</li>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/features/roadmap.html:42]:
```suggestion
    <li>Launch RCS messaging (<a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/service-manual/agile-delivery/how-the-alpha-phase-works">alpha</a>)</li>
```

File: app/templates/views/guidance/features/roadmap.html [hunk 2]
```diff
@@ -27,28 +27,30 @@ <h1 class="heading-large">Roadmap</h1>
   <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working on now</h2>
 
   <ul class="govuk-list govuk-list--bullet">
-    <li>Let services create letter templates that comply with the <a class="govuk-link govuk-link--no-visited-state" href="https://law.gov.wales/culture/welsh-language/welsh-language-act-1993">Welsh Language Act 1993</a></li>
-    <li>Improve our front-end code to meet our targets for accessibility</li>
     <li>Make it easier for users to join an existing service</li>
+    <li>Improve our front-end code to meet our targets for accessibility</li>
     <li>Let organisations approve new services (<a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/service-manual/agile-delivery/how-the-beta-phase-works">public beta</a>)</li>
-    <li>Let services set a custom sender name and email address</li>
-    <li>Update the Service Manual guidance about <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/service-manual/design/sending-emails-and-text-messages">planning and writing emails and text messages</a></li>
```
Comment (tombye) [inline app/templates/views/guidance/features/roadmap.html:35]: Can we please change this link so it wraps around the 'the Service Manual guidance about planning and writing emails and text messages' text instead?

**Review by tombye — CHANGES_REQUESTED** [review]
The code looks code and renders fine in a browser and also the links all work. I just have one comment but otherwise this looks good 👍🏻

File: app/templates/views/guidance/features/roadmap.html (hunk 2 above)
Comment (karlchillmaid) [reply app/templates/views/guidance/features/roadmap.html:35]: We’re removing this line anyway, so the link won’t be part of the updated page.

Comment (tombye) [reply app/templates/views/guidance/features/roadmap.html:35]: 🤦🏻

**Review by tombye — APPROVED** [review]
I only had one change requested and it was wrong 🤦🏻, so 👍🏻

**Pull request merged**